tests/example_app/example/page1/page1a/bp_page1a.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+
+bp_page1a = Blueprint('page_1a', __name__,
```

Comment:
Can you change all of the page, page1a, page1b and page 2 references to use more "real" names? I found it to be a little hard to follow with everything named like this.

I'd suggest something like:

- `page1 -> blog`
  - `page1a -> comments`
  - `page1b -> I'm not sure`
- `page2 -> about`

Feel free to pick something else for all of these as long as it's cohesive.

I noticed there's no tests for `page1b` and no digested static files after I ran the digest command, was this intentional?

Reply:
[e98d391 updated example app](https://github.com/nickjj/flask-static-digest/pull/33/commits/e98d3917c1f0ef23ef450c86c11f5b4f4dde4350)

Ended up going:
```
page1 -> about
    page1a -> our-story
    page1b -> our-team
page2 -> contact
```
Hope that makes the site structure more intuitive :)

Reply:
Thanks, it does.

Can you take it 1 step further and change `our-story` to `story` and `our-team` to `team`?

Reply:
[3174c64 example app code styling](https://github.com/nickjj/flask-static-digest/pull/33/commits/3174c64846fad1973666d27628e9f6cbc4ea9a04)